flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/Task.java

```diff
@@ -495,6 +496,11 @@ public boolean isCanceledOrFailed() {
 				executionState == ExecutionState.FAILED;
 	}
 
+	@Override
```

Comment:
This can be removed as previously commented. This logic can be covered by above `isBackPressured`